common/common.cpp

```diff
@@ -767,6 +767,15 @@ bool gpt_params_find_arg(int argc, char ** argv, const std::string & arg, gpt_pa
         params.hf_file = argv[i];
         return true;
     }
+    if (arg == "-hft" || arg == "--hf-token") {
+        params.hf_token = NULL;
```

Comment:
Where is hf_token definition?

Reply:
i cant find the struct gpt_params defination to add this.

Reply:
it's in common.h